app/javascript/hooks/queries/recordings/useRecordings.jsx

```diff
@@ -1,16 +1,11 @@
 import { useQuery } from 'react-query';
-import axios from 'axios';
+import { useSearchParams } from 'react-router-dom';
+import axios, { ENDPOINTS } from '../../../helpers/Axios';
 
-export default function useRecordings(input, setRecordings) {
-  return useQuery(['getRecordings', input], () => axios.get('/api/v1/recordings.json', {
-    params: {
-      search: input,
-    },
-    headers: {
-      'Content-Type': 'application/json',
-      Accept: 'application/json',
-    },
-  }).then((resp) => {
-    setRecordings(resp.data.data);
-  }));
+export default function useRecordings(input = '') {
+  const [params] = useSearchParams();
+  params.set('search', input);
```

Comment:
Is this required? Can we not just manually specificy `params: { search: input }` as we do above?

Reply:
I don't like it either, I'm changing it to something else.